src/envVariables.js

```diff
@@ -112,52 +135,79 @@ let enviromentVariables = {
 	REPORT_ISSUE_EMAIL_TEMPLATE_CODE: {
 		message: 'Required reported issue email template code',
 		optional: process.env.ENABLE_EMAIL_FOR_REPORT_ISSUE === 'true' ? false : true,
+		default: 'user_issue_reported',
+	},
+	BIB_BLUE_BUTTON_BASE_URL: {
+		message: 'Big blue button base url',
+		optional: process.env.DEFAULT_MEETING_SERVICE === 'OFF' ? true : false,
+		default: '/bigbluebutton/',
 	},
 	BIG_BLUE_BUTTON_SESSION_END_URL: {
 		message: 'Big blue button session end url.',
-		optional: false,
+		optional: process.env.DEFAULT_MEETING_SERVICE === 'OFF' ? true : false,
 	},
 	ERROR_LOG_LEVEL: {
 		message: 'Required Error log level',
-		optional: false,
+		optional: true,
+		default: 'silly',
 	},
 	DISABLE_LOG: {
 		message: 'Required disable log level',
-		optional: false,
+		optional: true,
+		default: false,
 	},
 	DEFAULT_MEETING_SERVICE: {
```

Comment:
Can we make this as optional and have default value as 'OFF'

Reply:
no we cannot.

Reply:
Why not?

Reply:
here i gave default as OFF and it did not take in the next upcomming conditions which i have used 
so i need to mention int he env file so other conditions work.